man/pdfpc.in

```diff
@@ -32,6 +32,10 @@ monitors).
 .BI "\-B, \-\-list\-bindings"
 List action bindings defined
 .TP
+.BI "\-c, \-\-cfg\-statement"=STRING
```

Comment:
I propose to allow multiple statement-flags on the command line and not one with some delimiter. This also reduces the pain of maybe escaping/unescaping some characters in the string.

If I read https://valadoc.org/glib-2.0/GLib.OptionContext.html correctly, then it's possible to tag an option to have multiple occurrences.

Reply:
You're right, I missed it.